### PR TITLE
Feature/simplify installtion

### PR DESCRIPTION
**Pull request by mieszko4**
# Summary

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by mieszko4** [comment]
There is a lot of moving parts here, please someone read it through if that makes sense.
Note hidden sections

File: docs/installation.md [hunk 1]
```diff
@@ -183,248 +228,204 @@ In short, you would need to
...
-def DEFAULT_BUILD_TOOLS_VERSION             = "26.0.2"
-def DEFAULT_TARGET_SDK_VERSION              = 26
-def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION    = "12.0.1"
-def DEFAULT_SUPPORT_LIBRARY_VERSION         = "27.1.0"
+```xml
+<application ...>
+...
+  <meta-data
+      android:name="com.google.firebase.ml.vision.DEPENDENCIES"
+      android:value="ocr, face" /> <!-- choose models that you will use -->
```
Comment (mieszko4) [inline docs/installation.md]: I did not find barcode model on mlkit docs

File: docs/installation.md [hunk 2]
```diff
@@ -183,248 +228,204 @@ In short, you would need to
...
   defaultConfig {
     ...
-    missingDimensionStrategy 'react-native-camera', 'general' <-- insert this line
+    missingDimensionStrategy 'react-native-camera', 'mlkit' // <--- replace general with mlkit
   }
 }
 ```
 
-or, if using MLKit for text/face/barcode recognition:
+### Setting up Firebase
```
Comment (mieszko4) [inline docs/installation.md]: For android I actually managed to make barcode recognition working without installing firebase. Not sure why.
I used rn `0.61.4` with `npx react-native init`

**Comment by sibelius** [comment]
it improved a lot the docs, tks

**Pull request merged**